ci/deps/travis-36-cov.yaml

```diff
@@ -14,14 +14,15 @@ dependencies:
   - matplotlib
   - nomkl
   - numexpr
-  - numpy
+  - numpy=1.15.*
   - openpyxl
+  - pandas-gbq
```

Comment:
I added a pinned version in `travis-36-locale` and an unpinned one here (as this had fallen off the radar completely)